CONTRIBUTING.rst

```diff
@@ -213,3 +209,5 @@ Flocker bugs should normally be :ref:`reported publicly<talk-to-us>`, but due to
 Instead, if you believe you have found something in Flocker (or any other ClusterHQ software) which has security implications, please send a description of the issue via email to security@clusterhq.com.
 Your message will be forwarded to the ClusterHQ security team (a small group of trusted developers) for triage and it will not be publicly readable.
 Once you have submitted an issue via email, you should receive an acknowledgment from a member of the security team within 48 hours, and depending on the action to be taken, you may receive further follow up emails.
+
+.. _JIRA: https://clusterhq.atlassian.net/secure/Dashboard.jspa
```

Comment:
Not that I can see in an obvious way - what I have done is removed an instance of the duplication (it was unnecessarily twice in one document) so that the link is only in two places.